ipld/plugin/nmt.go

```diff
@@ -44,6 +44,10 @@ const (
 	namespaceSize = 8
 	// nmtHashSize is the size of a digest created by an NMT in bytes.
 	nmtHashSize = 2*namespaceSize + sha256.Size
+
+	// mhOverhead is the size of the prepended buffer of the CID encoding
+	// for NamespacedSha256.
+	mhOverhead = 4
```

Comment:
I think it would be good if this was explained a bit further in the sense that it is clear how these are 4 bytes (a simple 4 = fooFormat + bar + foobar = 2+1+1 or sth like that).

Reply:
I don't think we need to explain how multihashes work and instead we can direct to their docs if available.

Reply:
Yeah, that is also OK. As long as the reader can learn about why it's 4.